### DISCUSSION TITLE
Workflow

### DISCUSSION THREAD

**Discussion by codetricity** [theta360developers/thanks#14] [Announcements]
improve workflow

using Sparrow.  Put tips here.

**Comment by AirTechWick** [comment]
Should I build the files or @codetricity and @jcasman build the files? Not sure if my workflow is making me do an extra commit or if it's the way I should be doing it?

**Comment by jcasman** [comment]
Based on discussion in office, @erik will not run npm run live. No need.

**Reply by codetricity** [reply]
I agree.  If you run live, it may interfere with the merge
